bin/hzz2l2v/runHZZ2l2vAnalysis.cc

```diff
@@ -62,6 +62,98 @@
 
 using namespace std;
 
+namespace utils
+{
+    namespace cmssw
+    {
+        
+        
+        std::vector<double> smearJER(double pt, double eta, double genPt)
+        {
+            std::vector<double> toReturn(3,pt);
+            if(genPt<=0) return toReturn;
+            
+            // FIXME: These are the 8 TeV values.
+            //
+            eta=fabs(eta);
+            double ptSF(1.0), ptSF_err(0.06);
+            if(eta<0.8)                  { ptSF=1.061; ptSF_err=sqrt(pow(0.012,2)+pow(0.023,2)); }
```

Comment:
here only the second number has been updated (0.023)
I did not understand from where the first number comes form, so to be conservative I kept the same as in run 1
(it is true also for the following lines